app/views/docs/databases.phtml

```diff
@@ -226,16 +225,47 @@ func main() async throws {
     </li>
 </ul>
 
+<h2><a href="/docs/databases#indexes" id="indexes">Indexes</a></h2>
+<p>
+    Databases use indexes to quickly locate data without having to search through every document for matches.
+    To ensure the best performance, Appwrite requires an index for every query.
+    You can create an index by navigating to your collection's <b>Indexes</b> tab or by using your favorite Server SDK. If you plan to query multiple attributes in a single query, you will need an index with <b>all</b> queried attributes.
+</p>
+
+<p>
+    The following indexes are currently supported:
+</p>
+<table cellspacing="0" cellpadding="0" border="0" class="full margin-bottom-large">
+    <thead>
+        <tr>
+            <th style="width: 250px">Type</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>key</code></td>
+            <td>Plain Index to allow queries.</td>
+        </tr>
+        <tr>
+            <td>unique</td>
+            <td>Unique Index to disallow duplicates.</td>
+        </tr>
+        <tr>
+            <td><code>fulltext</code></td>
+            <td>For searching within string attributes</td>
+        </tr>
+    </tbody>
+</table>
 
-<h3><a href="/docs/databases#querying-documents" id="querying-documents">Querying Documents</a></h3>
+<h2><a href="/docs/databases#retrieve" id="retrieve">Retrieving Documents</a></h2>
+<p>Documents can be retrieved using the <a href="/docs/client/databases#databasesListDocument">List Document</a> endpoint. Results can be filtered, sorted, and paginated using Appwrite's shared set of query methods. You can find a full guide on querying in the <a href="/docs/queries">Queries Guide</a>.</p>
```

Comment:
You can now also use `select` queries to select only a subset of a documents attributes. Doing so can improve query performance and API request time as less data is fetched/transferred